deploy/early-access/reference/databases.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Databases
+description: Connect to external database intances and integrate your applications and their environments seamlessly.
+---
+
+:::info
+
+You are viewing the documentation for Deno Deploy<sup>EA</sup>. Looking for
+Deploy Classic documentation? [View it here](/deploy/).
+
+:::
+
+Connect your applications to external databases like PostgreSQL, MySQL, MongoDB,
+and more. Deno Deploy automatically provisions isolated databases for your
+different deployment environments.
+
+## Overview
```

Comment:
Nit: Wondering if we should drop this bit as it's sorta the same as the first paragraph of the first "Overview" paragraph.